app/controllers/model-types.js

```diff
@@ -3,27 +3,36 @@ import { get, computed } from '@ember/object';
 import LocalStorageService from 'ember-inspector/services/storage/local';
 import { sort } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
+import { HIDE_EMPTY_MODELS_KEY, ORDER_MODELS_BY_COUNT_KEY } from 'ember-inspector/utils/local-storage-keys';
 
 export default Controller.extend({
   application: controller(),
   navWidth: 180,
-  sortProperties: ['name'],
+  sortByNameProp: Object.freeze(['name']),
```

Comment:
I'm curious what this `Object.freeze` is for?

Reply:
Sure I can make those changes. There prob should be some standardize, see [Ember.sort](https://www.emberjs.com/api/ember/release/functions/@ember%2Fobject%2Fcomputed/sort) docs using Object.Freeze

Reply:
Updated